posthog/session_recordings/session_recording_api.py

```diff
@@ -612,29 +612,17 @@ def snapshots(self, request: request.Request, **kwargs):
         NB version 1 of this API has been deprecated and ClickHouse stored snapshots are no longer supported.
         """
 
-        recording: SessionRecording = self.get_object()
+        timer = ServerTimingsGathered()
+
+        with timer("get_recording"):
+            recording: SessionRecording = self.get_object()
 
         if not SessionReplayEvents().exists(session_id=str(recording.session_id), team=self.team):
             raise exceptions.NotFound("Recording not found")
 
         source = request.GET.get("source")
         is_v2_enabled = request.GET.get("blob_v2", "false") == "true"
 
-        event_properties = {
-            "team_id": self.team.pk,
-            "request_source": source,
-            "session_being_loaded": recording.session_id,
-        }
-
-        if request.headers.get("X-POSTHOG-SESSION-ID"):
-            event_properties["$session_id"] = request.headers["X-POSTHOG-SESSION-ID"]
-
-        posthoganalytics.capture(
```

Comment:
fly-by remove these capture calls, they're noisy and we don't  barely reference them

_and_ we have prom counters for the same data